prefect_databricks/flows.py

```diff
@@ -255,14 +256,45 @@ async def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_pa
     multi_task_jobs_runs_id = multi_task_jobs_runs["run_id"]
 
     seconds_waited_for_run_completion = 0
+
+    job_status_info = {}
+    task_status_info = {}
+
     while seconds_waited_for_run_completion <= max_wait_seconds:
         jobs_runs_metadata_future = await jobs_runs_get.submit(
             run_id=multi_task_jobs_runs_id,
             databricks_credentials=databricks_credentials,
             wait_for=[multi_task_jobs_runs_future],
         )
         jobs_runs_metadata = await jobs_runs_metadata_future.result()
+        jobs_runs_run_id = jobs_runs_metadata.get("run_id", "")
+        jobs_runs_run_page_url = jobs_runs_metadata.get("run_page_url", "")
         jobs_runs_state = jobs_runs_metadata["state"]
+
+        __log_state(
+            job_status_info,
+            jobs_runs_run_id,
+            jobs_runs_state,
+            jobs_runs_run_page_url,
+            logger,
+            False,
+        )
```

Comment:
I think it'd simplify the flow code if the run metadata was passed directly in to the `__log_state` function and it pulled what it needed off the run metadata object. You could replace `jobs_runs_run_id`, `jobs_runs_state`, and `jobs_runs_run_page_url` with one argument.